src/index.js

```diff
@@ -0,0 +1,183 @@
+const state ={
+    temp: 0,
+    number: document.getElementById("temperature-now"),
+}
+
+const upButton = document.getElementById("up-button");
+const downButton = document.getElementById("button-down");
+const tempButton = document.getElementById("get-temp")
+// let number = document.getElementById("temperature-now");
+// let temp = parseFloat(number.innerText);
+state.temp = parseFloat(state.number.innerText);
+const weatherGarden = document.getElementById('weather-garden');
+const weatherEmojis = document.getElementById('weather-emojis');
+const cityDisplay = document.getElementById("city-display")
+const cityId = document.getElementById("city-input")
+const skyOptions = document.getElementById('change-sky');
+const skyEmojis = document.getElementById('chosen-sky-emojis');
+const resetBtn = document.getElementById('resetbtn');
```

Comment:
Pieces of code like this are usually found in an event handler that runs on the event DOMContentLoaded, basically once the page is html is loaded on the webpage you will run a bunch of initial logic such as grabbing elements and adding event handlers to them. It will look something like this:

```javascript 
document.addEventListener("DOMContentLoaded", function () {
    const increaseTemp = document.querySelector("#increase-temp");
    const decreaseTemp = document.querySelector("#decrease-temp");
    const displayTemp = document.querySelector("#display-temp");
    const resetButton = document.querySelector("#reset-button"); 
    const searchButton = document.querySelector("#search-button");
    const cityName = document.getElementById("city-name");
    const cityInput = document.getElementById("city-input");
    const selectSky = document.querySelector("#sky-dropdown");
    const result = document.querySelector("#sky");
    const landscape = document.querySelector("#landscape");
    
    increaseTemp.addEventListener("click", increaseTemperature);
    decreaseTemp.addEventListener("click", decreaseTemperature);
    ...
}
```

This is also usually found at the bottom of the file too, that way you can put all your function definitions for event handlers above it!